models/study_dfs.py

```diff
@@ -98,46 +98,3 @@ def dynamical_query(all_advance_query):
 
     search_final_query = search_final_query + ";"
     return search_final_query
-
-
-def get_general_info(studyId, conn):
```

Comment:
For history clarity, this is now omitted or replace with an alternative way?

Reply:
The "general info" that is fetched is for metabolites or strains of the study. Because we now have an ORM object, fetching the metabolites is just `study.metabolites`. In the view, when we loop through them, `for metabolite in study.metabolites` triggers the SQL query.

So, instead of defining a function that fetches all the data that is needed in the page, we have an object graph and we request from it the study's relationships. I'll document this at a later time. It's also important to be careful about N+1 queries, but probably not for this page.

Reply:
No need! I was already aware but I thought this is a great example of how useful `ORM` can be! :tada: